tests/locust/generated_from_requests.py

```diff
@@ -0,0 +1,34 @@
+# This script was generated by Taurus
+
+from gevent import sleep
+from re import findall, compile
+from locust import HttpLocust, TaskSet, task
+
+class UserBehaviour(TaskSet):
+    @task(1)
+    def generated_task(self):
+        with self.client.get(url="/", headers={"var2": "val2"}, timeout=30, catch_response=True) as response:
+            if not all(str(val) in response.content for val in ['text1', 'text2']):
+                response.failure("['text1', 'text2'] not found in body")
```

Comment:
Maybe we should generate one if-clause per assertion? This way Locust will produce more specific error messages (not `['text1', 'text2'] not found in body`, but `text1 not found in body` or `text2 not found in body`).